### PR TITLE
Fix minor typo.

### PR DESCRIPTION
**Pull request by delputnam**

### CODE REVIEW

File: client/my-sites/domains/domain-management/edit/transfer.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ class Transfer extends React.PureComponent {
 			noCancelNotice = (
 				<Notice status={ 'is-info' } showDismiss={ false }>
 					{ translate(
-						'This transfer has been started is waiting authorization from your current provider. ' +
+						'This transfer has been started and is awaiting authorization from your current provider. ' +
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/transfer.jsx]: Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'This transfer has been started is waiting authorization from your current provider. If you need to cancel the transfer, please contact them for assistance.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (aidvu) [reply client/my-sites/domains/domain-management/edit/transfer.jsx]: I'd go with `is waiting for`. Your call.

Comment (delputnam) [reply client/my-sites/domains/domain-management/edit/transfer.jsx]: Yeah. That sounds more like the correct voice.

File: client/my-sites/domains/domain-management/edit/transfer.jsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ class Transfer extends React.PureComponent {
 			noCancelNotice = (
 				<Notice status={ 'is-info' } showDismiss={ false }>
 					{ translate(
-						'This transfer has been started is waiting authorization from your current provider. ' +
+						'This transfer has been started and is waiting for authorization from your current provider. ' +
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/transfer.jsx]: Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'This transfer has been started is waiting authorization from your current provider. If you need to cancel the transfer, please contact them for assistance.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**